src/SerializeAST.cc

```diff
@@ -795,16 +803,16 @@ sos::Object WrapBlueprintAST(const Blueprint& blueprint)
 
     // Resource Groups
     blueprintObject.set(SerializeKey::ResourceGroups,
-                        WrapCollection<Element>()(blueprint.content.elements(), WrapResourceGroup, IsElementResourceGroup, false));
+                        WrapCollectionIf<Element>()(blueprint.content.elements(), WrapResourceGroup, IsElementResourceGroup, context));
 
     // Content
     blueprintObject.set(SerializeKey::Content,
-                        WrapCollection<Element>()(blueprint.content.elements(), WrapElement));
+                        WrapCollection<Element>()(blueprint.content.elements(), WrapElement, context));
 
     return blueprintObject;
 }
 
-sos::Object drafter::WrapBlueprint(const snowcrash::ParseResult<snowcrash::Blueprint>& blueprint, const bool expandMSON)
+sos::Object drafter::WrapBlueprint(const snowcrash::ParseResult<snowcrash::Blueprint>& blueprint, ConversionContext& context, const bool expandMSON)
```

Comment:
Please move the `expandMSON` argument and `ExpandMSON` global variable into context.